appcircle_ios_flutter_testinium/1.0.5/component.yaml

```
@@ -0,0 +1,86 @@
+platform: iOS
+buildPlatform: Flutter
+displayName: Testinium
+description: "Enables uploading mobile applications to Testinium and running the selected test plan."
+docsUrl: https://docs.appcircle.io/workflows/common-workflow-steps/testinium
+requiredComponents: "appcircle_ios_flutter_build,appcircle_ios_build_sign"
+precedingComponents:
+followingComponents:
+webUrl: https://github.com/appcircleio/appcircle-testinium-component
+repoUrl: https://github.com/appcircleio/appcircle-testinium-component.git
+commit: d08fa3f
+inputs:
+- key: "AC_TESTINIUM_APP_PATH"
+  defaultValue: "$AC_TESTINIUM_APP_PATH"
+  isRequired: true
+  title: Path of the build
+  description: "Full path of the build. For example `$AC_EXPORT_DIR/Myapp.ipa`."
```

Export path will be generated after export build artifact. We have rule running after xcodebuild for devices. All artifacts send output direction first.
```suggestion
  description: "Full path of the build. For example `$AC_OUTPUT_DIR/MyApp.ipa`."
```